File: kyonggiyo-core/src/main/java/kyonggiyo/domain/auth/exception/InvalidTokenException.java
package kyonggiyo.domain.auth.exception;

import kyonggiyo.common.exception.AuthenticationException;
import kyonggiyo.common.exception.ErrorCode;

public class InvalidTokenException extends AuthenticationException {

    public InvalidTokenException(ErrorCode errorCode) {
        super(errorCode);
    }

    public InvalidTokenException(ErrorCode errorCode, String loggingMessage) {
        super(errorCode, loggingMessage);
    }

}

File: kyonggiyo-core/src/main/java/kyonggiyo/application/auth/domain/exception/InvalidTokenException.kt
package kyonggiyo.application.auth.domain.exception

import kyonggiyo.common.exception.AuthenticationException
import kyonggiyo.common.exception.ErrorCode

class InvalidTokenException : AuthenticationException {
    constructor(errorCode: ErrorCode?) : super(errorCode!!)
    constructor(errorCode: ErrorCode?, loggingMessage: String?) : super(errorCode!!, loggingMessage!!)
}
